jib-gradle-plugin/CHANGELOG.md

```diff
@@ -14,11 +14,14 @@ All notable changes to this project will be documented in this file.
 - Container creation date set to timestamp 0 ([#341](https://github.com/GoogleContainerTools/jib/issues/341))
 - Does not authenticate base image pull unless necessary - reduces build time by about 500ms ([#414](https://github.com/GoogleContainerTools/jib/pull/414))
 - `jvmFlags`, `mainClass`, `args`, and `format` are now grouped under `container` configuration object ([#384](https://github.com/GoogleContainerTools/jib/issues/384))
+- Warns instead of errors when classes not found ([#462](https://github.com/GoogleContainerTools/jib/pull/462))
```

Comment:
Do we need to do this in the maven plugin as well?

Reply:
Nah, since jib-maven-plugin does not automatically depend on `compile`. Also, it just looks at build output directories, so it's not tied to some `java` plugin like Gradle.